spec/presenters/person_presenter_spec.rb

```diff
@@ -1,7 +1,8 @@
 # frozen_string_literal: true
 
 describe PersonPresenter do
-  let(:profile_user) { FactoryGirl.create(:user_with_aspect) }
+  let(:profile_user) { FactoryGirl.create(:user_with_aspect,
+    profile: FactoryGirl.create(:profile_with_image_url)) }
```

Comment:
- Layout/AlignParameters: Align the parameters of a method call if they span more than one line.
- Layout/BlockEndNewline: Expression at 5, 59 should be on its own line.